testsuite/analyze_energy.py

```diff
@@ -3,17 +3,22 @@
 import unittest as ut
 import numpy as np
 import espressomd
+from espressomd import electrostatics
 from espressomd.interactions import HarmonicBond
 
 
-@ut.skipIf(not espressomd.has_features("LENNARD_JONES"), "Skipped because LENNARD_JONES turned off.")
+features = ["LENNARD_JONES", "ELECTROSTATICS", "P3M"]
```

Comment:
Please move the p3m initialization and import to the test_electrostatics function and set the feature check there, so the rest of the test can run w/o electrostatics.